lib/jnpr/junos/facts/routing_engines.py

```diff
@@ -38,7 +41,7 @@ def facts_routing_engines(junos, facts):
                 re_fd[factoid.replace('-', '_')] = x_f.text
 
         if 'mastership_state' in re_fd:
-            if facts[re_name]['mastership_state'] == 'master':
+            if facts[re_name]['mastership_state'].find('master') > 0:
```

Comment:
Hi @vnitinv ,

It's been so long I don't recall the reasoning for this change. Feel free to ignore it. ;)